WORKSPACE

```diff
@@ -33,6 +33,14 @@ load("@bazel_gazelle//:deps.bzl", "gazelle_dependencies", "go_repository")
 
 gazelle_dependencies()
 
+# Load org.golang/x/sys repository here as a workaround to fix issue #1876.
```

Comment:
I think you should be more verbose in this comment. I would expect to see an explanation like the comment https://github.com/google/gvisor/issues/1876#issuecomment-588544324 .

And I think you need to add TODO(gvisor.dev/issue/1876): at the beginning of this comment.

Reply:
Done. Thanks!